tests/Unit/PointwiseFunctions/AnalyticSolutions/ForceFree/Test_AlfvenWave.cpp

```diff
@@ -0,0 +1,81 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <memory>
+#include <random>
+#include <utility>
+
+#include "Framework/CheckWithRandomValues.hpp"
+#include "Framework/SetupLocalPythonEnvironment.hpp"
+#include "Framework/TestCreation.hpp"
+#include "Framework/TestHelpers.hpp"
+#include "Helpers/DataStructures/MakeWithRandomValues.hpp"
+#include "Parallel/RegisterDerivedClassesWithCharm.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/ForceFree/AlfvenWave.hpp"
+#include "PointwiseFunctions/InitialDataUtilities/InitialData.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace {
+
+using InitialData = evolution::initial_data::InitialData;
+using AlfvenWave = ForceFree::Solutions::AlfvenWave;
+
+struct AlfvenWaveProxy : AlfvenWave {
+  using AlfvenWave::AlfvenWave;
+  using variables_tags =
+      tmpl::list<ForceFree::Tags::TildeE, ForceFree::Tags::TildeB,
+                 ForceFree::Tags::TildePsi, ForceFree::Tags::TildePhi,
+                 ForceFree::Tags::TildeQ>;
+
+  tuples::tagged_tuple_from_typelist<variables_tags> return_variables(
+      const tnsr::I<DataVector, 3>& x, double t) const {
+    return this->variables(x, t, variables_tags{});
+  }
+};
+
+SPECTRE_TEST_CASE(
+    "Unit.PointwiseFunctions.AnalyticSolutions.ForceFree.AlfvenWave",
+    "[Unit][PointwiseFunctions]") {
+  // test creation
+  const auto solution =
+      TestHelpers::test_creation<AlfvenWave>("WaveSpeed: 0.5");
+  CHECK(solution == AlfvenWave(0.5));
+  CHECK(solution != AlfvenWave(-0.5));
+
+  // test serialize
+  test_serialization(solution);
+
+  // test move
+  test_move_semantics(AlfvenWave{0.5}, AlfvenWave{0.5});
+
+  // test derived
+  Parallel::register_classes_with_charm<AlfvenWave>();
+  const std::unique_ptr<InitialData> initial_data_ptr =
+      std::make_unique<AlfvenWave>();
```

Comment:
Don't use a default-constructed object here, and then check that the deserialized object is equal to the original.